web/src/components/filings-calendar/FilingsCalendarTaxAccess.tsx

```diff
@@ -40,7 +41,21 @@ export const FilingsCalendarTaxAccess = (): ReactElement => {
           taxId: userData.profileData.taxId as string,
           encryptedTaxId: userData.profileData.encryptedTaxId as string,
         });
-        updateQueue?.queue(updatedUserData).update();
+
+        let record;
+
+        if (updatedUserData.profileData.municipality?.name) {
```

Comment:
[sand] we should pull this behavior into a shared function so we don't have to maintain it twice